notebooks/ceo_area_analysis.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<p>I think instead of these comments you can just print out <code>Number of times consensus label 'mistaken' for a different label"</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>